src/dotnet/APIView/APIViewWeb/Pages/Shared/_ReviewsPartial.cshtml

```diff
@@ -46,7 +46,24 @@
                                             <img class="mx-1" src="~/icons/go-original.svg" alt="@review.Language">
                                             break;
                                         case "java":
-                                            <img class="mx-1" src="~/icons/java-original.svg" alt="@review.Language">
+                                            if(@review.Variant != null)
+                                            {
+                                                switch(@review.Variant.ToLower())
+                                                {
+                                                    case "spring":
+                                                        <img class="mx-1" src="~/icons/java-spring-original.svg" alt="@review.Language @review.Variant">
+                                                        break;
+                                                    case "android":
+                                                        <img class="mx-1" src="~/icons/java-android-original.svg" alt="@review.Language @review.Variant">
+                                                        break;
+                                                    default:
+                                                        <img class="mx-1" src="~/icons/java-original.svg" alt="@review.Language">
+                                                        break;
+                                                }
+                                            } else
```

Comment:
As much as it pains me (as a Java engineer), the C# expectation here is to put the closing brace and the `else` on separate lines.